tests/_testlib.livecodescript

```diff
@@ -306,27 +306,41 @@ on TestRepeat pDesc, pHandler, pTarget, pTimeOut, pParamsArray
         	put ", pParamsArray[" & tParam & "]" after tParams
         end if
     end repeat
-    
+
     put tParams after tDoString
     put "; put the result into tResult" after tDoString
-    
+
     put false into tResult
-    
+
     local tTimer, tTimeout
     put false into tTimeout
     put the millisecs into tTimerStart
-    
+
     repeat while tResult is false and not tTimeout
        wait 1 millisecs with messages
 	   do tDoString
        if the millisecs - tTimerStart > pTimeOut then
           put true into tTimeOut
        end if
 	end repeat
-	
+
     TestAssert pDesc, tResult
-    
+
     if (not tResult) and tTimeOut then
 		TestDiagnostic pDesc & "- timed out"
 	end if
 end TestRepeat
+
+on TestRunStack pOptions, pStackFilePath
+	local tEnginePath
+	set the itemDelimiter to ":"
+	put item 2 of the address into tEnginePath
+
+	local tCommand
+	put format("\"%s\" %s \"%s\"", tEnginePath, pOptions, pStackFilePath) into tCommand
+
+	local tOutput
+	put shell(tCommand) into tOutput
```

Comment:
Either here or in the test you'll need to make sure that `-ui` is in the options if "command line" is in the enviroment, otherwise this will fail on Travis, I think (similar to the problem I've just had with standalone builder tests!)